packages/tracing/src/transaction.ts

```diff
@@ -64,6 +65,13 @@ export class Transaction extends SpanClass implements TransactionInterface {
     this._measurements = { ...measurements };
   }
 
+  /**
+   *
+   */
+  public setContexts(contexts: Contexts): void {
+    this._contexts = contexts;
+  }
```

Comment:
Unsure if there's a better way for this.

Reply:
cc @lobsterkatie @HazAT

Reply:
I've decided to remove this altogether.